Watchman.Discord/Areas/Users/BotCommands/Warns/WarnsCommand.cs

```diff
@@ -7,6 +7,6 @@ public class WarnsCommand : IBotCommand
     {
         [Optional]
         [UserMention]
-        public ulong User { get; set; }
+        public ulong? User { get; set; }
```

Comment:
czemu jako nullable?
jak ktoś nie ustawi to po prostu będzie 0

nullable lepiej używać w sytuacjach, gdzie jest różnica między null (nie istnieje) a 0

w tym przypadku 0 jest równoznaczne z nie istnieje
bo istniejący użytkownik nie może mieć ID == 0